internal/executor/options.go

```diff
@@ -45,9 +44,9 @@ func WithContainerOptions(containerOptions options.ContainerOptions) Option {
 	}
 }
 
-func WithContainerBackend(containerBackend containerbackend.ContainerBackend) Option {
+func WithContainerBackend(containerBackendType string) Option {
```

Comment:
Should be probably called `WithContainerBackendType()` now since we're only passing the type and not the instance.

Reply:
Right! See [708595a](https://github.com/cirruslabs/cirrus-cli/pull/525/commits/708595a2f40b55e3c93af37ebc48936adc22c345)